test/e2e/reconciler/scheduling/api_compatibility_test.go

```diff
@@ -58,14 +58,14 @@ func TestSchedulingOnSupportedAPI(t *testing.T) {
 		framework.WithSyncTargetName(firstSyncTargetName),
 		framework.WithSyncedUserWorkspaces(userWS),
 		framework.WithAPIExports(""),
-	).Start(t)
+	).Create(t).StartAPIImporter(t).StartHeartBeat(t)
 
 	secondSyncTargetName := fmt.Sprintf("secondsynctarget-%d", +rand.Intn(1000000))
 	t.Logf("Creating a SyncTarget with global kubernetes APIExports and syncer in %s", locationPath)
 	_ = framework.NewSyncerFixture(t, source, locationPath,
 		framework.WithSyncTargetName(secondSyncTargetName),
 		framework.WithSyncedUserWorkspaces(userWS),
-	).Start(t)
+	).Create(t).StartAPIImporter(t).StartHeartBeat(t)
```

Comment:
fix Logf above

Reply:
Fixed in commit 13a4a065c80f0bf3815d69defc0ffcd0728fce2d